pages/common/lp.md

```diff
@@ -0,0 +1,31 @@
+# lp
+
+> print files
+
+- List printers present on the machine.
+
+`lpstat -p | awk '{print $2}`
```

Comment:
I think the `lpstat -d` and `lpstat -p` commands should not be separate examples, but instead be mentioned in the description of the first "default printer" example and the first "named printer" example, respectively.